envelope.js

```diff
@@ -78,8 +78,8 @@ module.exports = function Envelope (keystore, state) {
     // TODO change unboxer signature to allow us to optionally pass variables
     // from key() down here to save computation
     const envelope = Buffer.from(ciphertext.replace('.box2', ''), 'base64')
-    const feed_id = new FeedId(author).toTFK()
-    const prev_msg_id = new MsgId(previous).toTFK()
+    const feed_id = bfe.encode(author)
+    const prev_msg_id = bfe.encode(previous)
```

Comment:
this is a really lovely simplification. Like this API
now I understand why you had to explicitly add null support